client/verta/tests/registry/conftest.py

```diff
@@ -1,8 +1,28 @@
 # -*- coding: utf-8 -*-
 
+import calendar
+import collections as collecs
+import datetime as dt
+import json
+from typing import Type, Union
+
+import boto3
+import click
+import cloudpickle as cp
+import numpy
+import pandas as pd
 import pytest
+import requests
+import sklearn
+import spacy
+import torch
+import urllib3
+import yaml
+from google.protobuf import message as msg
+from PIL import UnidentifiedImageError
```

Comment:
I mentioned this in Slack, but just to codify it in a code review: We should use `pytest.importorskip()` for these rather than `import`. Otherwise `pytest` will fail at collection time due to non-installed libraries.

Something that might look a bit hacky but is acceptable imo for tests is to create a `dependency_testing_model()` fixture that calls `importorskip()`s, and defines (and returns) `class DependencyTestingModel` in its body.

Reply:
Working on that right now actually.

Reply:
Ok, let me know what you think of the latest implementation.